app/submitter/converter.py

```diff
@@ -74,6 +76,12 @@ def convert_answers(schema, questionnaire_store, routing_path, flushed=False):
         'questionnaire_id': metadata['questionnaire_id'],
     }
 
+    if metadata.get('channel'):
```

Comment:
Can the dict objects be set to None and then ignored downstream? If yes, the if statements could be removed here.

Reply:
I think to keep it consistent it should only be added to the payload if it exists.

Reply:
For Census, channel should technically be a mandatory claim. However, that's simply because we have a requirement to send it downstream as opposed to runner needing it for any processing. It has been discussed with Data Architecture that channel should actually be a mandatory claim post census (i.e. all ONS submitted data should include the channel of launch.. that may well require a None value, or a JWT validation error at launch). However, at this point in the proceedings I'm happy to leave it as it is (i.e. required=false at eQ launch and then checked for its presence during submission). Runner will not break if it is not included. It is up to the calling system (e.g. RH or CC) to include it.

Reply:
Yes, it is an interesting case of us not caring about channel ourselves, but gatekeeping for downstream, proxying the rules further up the chain of custody. Arguably, the census schema should be the contract specifying what is essential and what is optional, agreed by all parties. The rules should probably not be internal to runner.